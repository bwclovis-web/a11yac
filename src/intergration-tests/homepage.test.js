import React from 'react';
import renderer from 'react-test-renderer';
import HomePage from '../pages/index';

describe("HomePage", () => {
    it('should render correctly ', () => {
        const tree = renderer.create(<HomePage/>).toJSON()
        expect(tree).toMatchSnapshot()
    });
})