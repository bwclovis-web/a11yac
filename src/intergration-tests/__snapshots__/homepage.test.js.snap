// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage should render correctly  1`] = `
Array [
  <header
    className="sc-bdfBwQ joBAPR"
  >
    <div
      className="container"
    >
      <h1 />
    </div>
  </header>,
  <div
    className="sc-dlfnbm cfIvPp"
  >
    <nav
      className="sc-gsTCUz lipnIV"
    >
      <ul>
        <li>
          <a
            href="/new-component"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            New Component
          </a>
        </li>
        <li>
          <a
            href="/component-listing"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            Current Components
          </a>
        </li>
      </ul>
    </nav>
    <main>
      <h1>
        Hi people
      </h1>
      <p>
        Welcome to your new Gatsby site.
      </p>
      <p>
        Now go build something great.
      </p>
      <p>
        <a
          href="/page-2/"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          Go to page 2
        </a>
         
        <br />
        <a
          href="/using-typescript/"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          Go to "Using TypeScript"
        </a>
      </p>
    </main>
  </div>,
]
`;
